packages/css/src/components/search-field/search-field.scss

```diff
@@ -37,28 +37,29 @@
   @include text-rendering;
   @include reset-input;
 
-  &:hover {
-    box-shadow: var(--ams-search-field-input-hover-box-shadow);
-  }
-
   &:focus {
     z-index: 1; // Make sure the focus outline isn’t cut off by the adjacent button
   }
 }
 
-.ams-search-field__input:invalid,
-.ams-search-field__input[aria-invalid="true"] {
+.ams-search-field__input::placeholder {
+  color: var(--ams-search-field-input-placeholder-color);
+  opacity: 100%; // This resets the lower opacity set by Firefox
```

Comment:
Should we apply the boy scout rule and extract a reset mixin for the 4 occurrences of this line?

Reply:
No, not shared, but we could create 4 local mixins. It is more code but it does express that these lines override UA styles, which we do for other declarations as well. And it would help not forget these when we get around to [share resets](https://gemeente-amsterdam.atlassian.net/browse/DES-1002).